real_time/surface_src/java_src/Alice/library/google-api-java-client/dependencies/css/maven-base.css

```diff
@@ -0,0 +1,151 @@
+body {
+  margin: 0px;
+  padding: 0px;

```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Values of 0 shouldn't have units specified.](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1282815935/issues/source?bid=2938665&fileBranchId=2945285#l3)